region/norfair/west.json

```diff
@@ -3164,7 +3152,7 @@
                 },
                 {
                   "name": "Croc Escape HiJump Freeze",
-                  "notable": true,
+                  "notable": false,
```

Comment:
This one needs a note.  Just a simple lure the Geruta left by jumping towards it from below, then freeze